pkg/network/node/egressip.go

```diff
@@ -57,10 +57,11 @@ type egressIPWatcher struct {
 	testModeChan chan string
 }
 
-func newEgressIPWatcher(localIP string, oc *ovsController) *egressIPWatcher {
+func newEgressIPWatcher(localIP string, oc *ovsController, informers common.SDNInformers) *egressIPWatcher {
```

Comment:
don't pass informers here, pass it to Start(); that way you don't have to create a fake-not-actually-usable informers arg to pass here in egressip_test.go (because egressip_test.go never calls Start(), because it just calls the update funcs by hand)